management/utils.py

```diff
@@ -179,6 +179,9 @@ def wait_for_service(port, public, env, timeout):
 				return False
 		time.sleep(min(timeout/4, 1))
 
+def get_php_version():
+	return "8.1"
```

Comment:
I hardcoded this here to have fewer places where the version is set. I think it is now reduced to the installation scripts (functions.sh) and this location.
Perhaps the python function can read PHP_VER from function.sh? I did not (yet?) think of a simple dependable way to do that.

Reply:
It would be nice if it can get the version from function.sh

I don't know either, how to do this best. Why not ask for technical advice on our Slack channel. As a team, we're happy to help each other ;)

Reply:
For now I don´t want to introduce additional complexity. I propose this for a future improvement.